source/app/powertabeditor.cpp

```diff
@@ -1762,6 +1762,22 @@ void PowerTabEditor::editViewFilters()
     }
 }
 
+void PowerTabEditor::zoomInScore()
+{
+    const int ZOOM_CHANGE_COEFITIENT = 25;
+    auto currentZoom = myDocumentManager->getCurrentDocument().getViewOptions().getZoom();
+
+    updateZoom(currentZoom + ZOOM_CHANGE_COEFITIENT);
+}
+
+void PowerTabEditor::zoomOutScore()
+{
+    const int ZOOM_CHANGE_COEFITIENT = 25;
```

Comment:
I don't have a good idea where this constant should live :/ I don't want to pollute the substantial `powertabeditor` class.
This constant is related to the "Zoom in shortcuts", something that we don't have a dedicated place for...

Reply:
I think I'd probably put that inside the `ViewOptions` class, maybe with just an increaseZoom / decreaseZoom method? If the predefined zoom levels (currently stored in `playbackwidget.ui` for populating the combobox widget) were instead stored in `ViewOptions`, the increase / decrease methods could also step through the predefined zoom levels rather than a fixed percentage